recipes/libs/botan.yaml

```diff
@@ -17,14 +17,26 @@ checkoutSCM:
     digestSHA1: e920c3531fa2daf7348e1a668bfdc0f446e10c8b
     stripComponents: 1
 
+checkoutScript: |
+    patchApplySeries $<@botan/*@>
+
 buildToolsWeak: [python3]
 buildVars: [AR, CXX, CXXFLAGS, CPPFLAGS, ARCH]
 buildScript: |
     mkdir -p build install
     pushd build
+
+    case "$AUTOCONF_HOST" in
+      *-l4re) target_os=l4re
+              stackprotector=--without-stack-protector
+              ;;
+      *) target_os=linux;;
+    esac
+
     $1/configure.py \
         --cpu=$ARCH \
-        --os=linux \
+        --os=$target_os \
+        $stackprotector \
```

Comment:
I guess this will not build any more for anything except L4Re. All scrips are running with `set -u`! I would suggest to `declare -a EXTRA_ARGS=( )` first and in case of L4Re do an `EXTRA_ARGS+=( --without-stack-protector )` and pass this to `configure.py` as `"${EXTRA_ARGS[@]}"`.

Reply:
Ack